week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,12 @@
 let groceriesToBuy = [];
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
```

Comment:
Do you think there is another way to assign values to the object properties outside the object?